### PR TITLE
changes.md: update for 1.6.7-tectonic.2 / 1.7.1-tectonic.1

### PR DESCRIPTION
**Pull request by Quentin-M**

### CODE REVIEW

File: changes.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Tectonic Changelog
 
-## Current
+## Tectonic 1.7.1-tectonic.1 (2017-08-07)
+
+* Updates to Kubernetes v1.7.1.
```
Comment (yifan-gu) [inline changes.md]: TCO allow to upgrade through every intermediate versions

**Comment by Quentin-M** [comment]
@alexsomesan I am waiting for input here - this is definitely not complete, just the skeleton. I asked the channel yesterday about it.

**Comment by alexsomesan** [comment]
Oh, didn't realise this was WIP. I'll revert the merged commit and you can reopen.

**Comment by alexsomesan** [comment]
reopened

**Pull request merged**